.gitignore

```diff
@@ -1 +1,2 @@
-node_modules
\ No newline at end of file
+node_modules
+queries.txt
```

Comment:
In your editor turn on "Insert Final Newline"
Reason: To have proper git difference
E.g. 
You old code was
```git
node_modules
```
You new code: (is having 2 diffs, although we only added 1 line )
```git
node_modules \n <----- New line character
queries.txt
```
Always have a new line at end of file so there's perfect git difference from next commit onwards. Applies to all files.

Reply:
Okay done Sir!